tech-far-hub/src/pages/get-started/template-index.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import type { HeadFC } from "gatsby";
+import { graphql, Link, PageProps } from "gatsby";
+import { CardGroup, Card, CardHeader, CardBody, CardFooter, Button } from "@trussworks/react-uswds";
+import Layout from "../../components/layout";
+
+const GetStartedPage: React.FC<PageProps<Queries.GetStartedPageQuery>> = ({
```

Comment:
Now that's some heavy duty typing right here! I want to get to a point where I can write types like this!